CBinding/Compiler/CCompiler.cs

```diff
@@ -33,75 +33,79 @@
 using System.IO;
 using System.Text;
 using System.CodeDom.Compiler;
+using System.Threading.Tasks;
 
 using MonoDevelop.Core;
 using MonoDevelop.Core.Execution;
 using MonoDevelop.Projects;
 using MonoDevelop.Ide;
+using System.Linq;
+
 
 namespace CBinding
 {
-	public abstract class CCompiler : ICompiler
+	public abstract class CCompiler : FolderBasedProject, ICompiler
 	{
 		protected string compilerCommand;
 		protected string linkerCommand;
-		
+		FilePath file;
```

Comment:
its was in CMakeProject.cs :- https://github.com/mhutch/cbinding/blob/master/CBinding/CMake/CMakeProject.cs#L65